cmake/f3dConfig.cmake.in

```diff
@@ -12,7 +12,7 @@
 #   f3d_MODULE_EXTERNAL_RENDERING  Will be enabled if F3D was built with external rendering support
 #   f3d_MODULE_RAYTRACING          Will be enabled if F3D was built with raytracing support
 #   f3d_BINDINGS_PYTHON            Will be enabled if F3D was built with python bindings
-#   f3d_SDK_PLUGIN                 Will be enabled if F3D plugin SDK is available
```

Comment:
is there a doc somewhere that needs to be udpated ?

Reply:
I couldn't find any doc

Reply:
added to the plugin issue

Reply:
https://f3d.app/doc/libf3d/PLUGINS.html

Reply:
it just do not mention the cmake var. well the doc is correct now then.